AutomatedTesting/Gem/PythonTests/Prefab/TestSuite_Main.py

```diff
@@ -113,7 +113,10 @@ class test_EditEntity_UnderImmediateInstance(EditorSharedTest):
 
     class test_EditEntity_UnderNestedInstance(EditorSharedTest):
         from .tests.overrides import EditEntity_UnderNestedInstance as test_module
-
+    
+    class test_AddEntity_UnderUnfocusedInstanceAsOverride(EditorSharedTest):
+        from .tests.overrides import AddEntity_UnderUnfocusedInstanceAsOverride as test_module
+        
```

Comment:
Test should be added to TestSuite_Periodic.py initially. After we validate it's running without issue in Jenkins for a few periodic runs, we can promote the test to Main.

Reply:
With this though, make sure you run a periodic_test_profile test run without any issues with the newly added test on internal Jenkins before submitting.